usecases/Concrete/knowledgeGraph/emodul/emodul_validation.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+from time import sleep
+from turtle import pos
+import requests
+import os
+import sys
+from pathlib import Path
+from git import Repo
+
+baseDir0 = Path(__file__).resolve().parents[0]
+baseDir1 = Path(__file__).resolve().parents[1]
+baseDir2 = Path(__file__).resolve().parents[2]
+ontologyPath = os.path.join(baseDir2,'ConcreteOntology')
+metadataPath = os.path.join(baseDir0,'E-modul-processed-data/emodul_metadata.csv')
+graphPath = os.path.join(baseDir0,'E-modul-processed-data/EM_Graph.ttl')
+#importedOntologiesPath = os.path.join(baseDir2,'GraphCreation/Ontologies')
+processedDataPath = os.path.join(baseDir0,'E-modul-processed-data')
+
+# clone and run rdfconverter docker
+Repo.clone_from('https://github.com/Mat-O-Lab/RDFConverter.git', os.path.join(baseDir1, 'RDFConverter'))
+subprocess.run(['docker-compose', '-f', os.path.join(baseDir1, 'RDFConverter/docker-compose.yml'), 'up', '-d', 'app'], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
```

Comment:
where is the docker-compose requirement set? IMO, that would be required in the conda environment?